src/python/py_image_generation_models.cpp

```diff
@@ -70,14 +70,26 @@ void init_clip_text_model(py::module_& m) {
     clip_text_model.def("get_config", &ov::genai::CLIPTextModel::get_config)
         .def("reshape", &ov::genai::CLIPTextModel::reshape, py::arg("batch_size"))
         .def("set_adapters", &ov::genai::CLIPTextModel::set_adapters, py::arg("adapters"))
-        .def("infer", &ov::genai::CLIPTextModel::infer, py::arg("pos_prompt"), py::arg("neg_prompt"), py::arg("do_classifier_free_guidance"))
+        .def("infer",
+            [](ov::genai::CLIPTextModel& self,
+                const std::string& pos_prompt,
+                const std::string& neg_prompt,
+                bool do_classifier_free_guidance
+            ) {
+                py::gil_scoped_release rel;
```

Comment:
can we use `py::call_guard<py::gil_scoped_release>()` for such cases to avoid explicit method implementation?
See https://pybind11.readthedocs.io/en/stable/advanced/misc.html#global-interpreter-lock-gil

Reply:
We do not make any python cast inside the function, so look like we can use mentioned by @ilya-lavrenov

Reply:
Agreed, done.